src/screens/Wallet.js

```diff
@@ -76,7 +85,55 @@ class Wallet extends React.Component {
     });
   }
 
+  /**
+   * Called when user clicks to unregister the token, then opens the modal
+   */
+  unregisterClicked = () => {
+    $('#unregisterModal').modal('show');
+  }
+
+  /**
+   * When user confirms the unregister of the token, hide the modal and execute it
+   */
+  unregisterConfirmed = () => {
+    $('#unregisterModal').modal('hide');
+    tokens.unregisterToken(this.props.selectedToken);
```

Comment:
Is it fast? Should we add an exception handler?

Reply:
Lib code:
```
  unregisterToken(uid) {
    const tokens = this.getTokens();
    const filteredTokens = tokens.filter((token) => token.uid !== uid);
    this.saveToStorage(filteredTokens);
    return filteredTokens;
  },  
```
Wallet code after lib code: (only updating with new array of tokens)
```
store.dispatch(newTokens({tokens, uid: hathorLib.constants.HATHOR_TOKEN_CONFIG.uid}));
```

In my opinion it's really fast and only local (no need for a loading) and no error possible. What do you think?

Reply:
The `saveToStorage` may be slow or fail for multiple reasons. We may return a promise or let the general exception handler report any problem.

Reply:
I've added the error handling when unregistering the token. Can you see if it's good?

Wallet commit: https://github.com/HathorNetwork/hathor-wallet/pull/44/commits/a7194bf589a6f2c935c3d16e863cd620f024472b
Lib commit: https://github.com/HathorNetwork/hathor-wallet-lib/pull/25/commits/2751ad630fa9382ad02bc09742aed7e21c6299bd

I will create a PR for the mobile app also now to add this handling.

Reply:
Great! Thanks!